playbook/buildfarmer/scripted_buildfarm_day.md

```diff
@@ -42,7 +42,17 @@ You can find the number of agents per buildfarm in [macines.md](../../machines.m
 If any agents are missing, you should: Ask the infrastructure team for a reason (e.g., some agents are down for maintenance) or manually check spot allocation problems in the AWS autoscaling group the missing agents are.
 
 
-## 4. Check failing jobs on buildfarm dashboards
+## 4. Check automation tools status
+
+Also, we have some automation tools that should run daily for the buildfarmer data to be updated. You can check the private repo readme and verify that all badges are green:
+
+* [osrf/buildfarm-tools-private](https://github.com/osrf/buildfarm-tools-private?tab=readme-ov-file#actions)
+
+If any of these badges are red, you should check the logs of the failed job and report the issue.
+
+These actions can also be manually triggered. Check [Automation Tools](./automation_tools.md) for more information.
+
+## 5. Check failing jobs on buildfarm dashboards
```

Comment:
Can you add in this list the standalone services dashboard? https://github.com/osrf/buildfarm-tools/blob/main/standalone_services.md

Same for the ultra scripted day.

Reply:
Now that you are here, it would be a great idea to add the dashboard of the macOS bottles here as well.